server/src/main/java/com/google/speech/tools/voxetta/services/DatastorePromptService.java

```diff
@@ -59,9 +60,13 @@ public boolean savePrompt(String type, String body) {
 
         promptEntity.setProperty("read", 0);
 
-        datastore.put(promptEntity);
+        try {
+            datastoreService.put(promptEntity);
+        } catch (DatastoreFailureException e) {
+            return new ErrorResponse(false, "Error: Unknown Datastore Failure Exception");
```

Comment:
nit: we could probably remove the `boolean` arg from `ErrorResponse` constructor. It's implied it will be success: false, so we could pass a hard-coded `false` into the super constructor.

No action required in this PR. just suggestion